app/graphql/client-only-query.ts

```diff
@@ -41,6 +44,13 @@ export default gql`
     countryCode @client
   }
 
+  query latLng {
+    latLng @client {
+      lat
+      lng
```

Comment:
I think we should also store the zoom. because otherwise we come back in a very "unzoomed" settings by default, and it feels almost like the settings was not saved as a result of it